fusor-ember-cli/app/controllers/undercloud-deploy.js

```diff
@@ -43,25 +48,21 @@ export default Ember.Controller.extend({
 
   actions: {
     deployUndercloud: function () {
-      var me = this;
```

Comment:
@sdherr, just a preference on my part, and to be consistent with the rest of the code to use `self`